templates/corporate/attribution.html

```diff
@@ -23,6 +23,11 @@ <h1>Website attributions</h1>
                     <img alt="" src="/static/images/landing-page/companies/software-engineer.svg" />
                     <p>"<a href="https://iconscout.com/illustration/software-engineer-2043023">Software engineer Illustration</a>" By <a href="https://iconscout.com/contributors/delesign/illustrations">Delesign Graphic</a> is licensed under <a href="https://creativecommons.org/licenses/by/4.0/">CC BY 4.0</a>.</p>
                 </li>
+                <li>
+                    <b>Default Integration Logo</b>
+                    <img alt="" src="/static/images/integrations/logos/generic.png" />
+                    <p>"<a href="https://www.flaticon.com/free-icons/technology">Technology icons"</a> created by Freepik - Flaticon.</p>
+                </li>
```

Comment:
Something seems off with quotation marks here.

Reply:
Yeah, @nagyem can you add a commit at the top of this PR correcting the quotation marks used here. We should be using `<p>"<a href='https://iconscout.com/illustration/software-engineer-2043023'>Software engineer Illustration</a>"`